tests-behat/grid.feature

```diff
@@ -5,7 +5,6 @@ Feature: Grid
     Given I am on "collection/grid.php"
     Then I search grid for "kingdom"
 #    make sure auto query trigger
-    And I wait for loading to start in "button.atk-search-button"
```

Comment:
@ibelar this is failing, is debounce really disabled?

Reply:
Yes, debounce is disable. You should probably call $this-jqueryWait() after $search->value($arg1) because the search will query the server with a callback and you are not before or after step.

Reply:
I already do https://github.com/atk4/ui/blob/6e749d0a02d2e1b928a4dd48212e6994c8bbbe0d/tests-behat/bootstrap/Context.php#L41 , that is why I am asking :)

Reply:
or probably because 'keyup' listener function is set on page load thus when setting debounce value afterward, it has no effect. Let me check...

Reply:
That is the reason, if we set debounce value prior to initializing the plugin then it is fine.

Suggestion:
- drop grid.feature since we now have crud.feature that does the same and more;
- in crud.feature we control the file so we can add this line of code: `$app->html->js(true, new JsExpression('atk.options.set("debounceTimeout", 20)'));` in _unit-test/crud.php file

Adding this line will make the test pass.

Reply:
`debounceTimeout` should be resolved INSIDE callback, thus https://github.com/atk4/ui/blob/develop/js/src/agile-toolkit-ui.js#L25 , so value should be resolved once called.

The debounce code is here: https://github.com/component/debounce/blob/master/index.js , maybe be can tweak the response more or completely copy and modify that impl.

Reply:
`jqueryWait` waits at least 20 ms, why do we need `iWaitForLoadingToStartIn`, what causes such large delay before `jQuery.active` is non-zero, eg. detectable?